src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -940,6 +940,20 @@ internal string GetPropertyRawText()
         /// <exception cref="ObjectDisposedException">
         ///   The parent <see cref="JsonDocument"/> has been disposed.
         /// </exception>
+        public void WriteAsProperty(string propertyName, Utf8JsonWriter writer)
+            => WriteAsProperty(propertyName.AsSpan(), writer);
+
+        /// <summary>
+        ///   Write the element into the provided writer as a named object property.
+        /// </summary>
+        /// <param name="propertyName">The name for this value within the JSON object, as UTF-16 text.</param>
```

Comment:
Since it's a string, the encoding shouldn't be mentioned.  So ditch everything after the comma.

Reply:
(Or, thanks to the way the diff lined up, copy the one for `ROS<char>`, which similarly doesn't need to mention the encoding, since it's not bytes)

Reply:
I was following the format in the writer APIs, where `ROS<byte>` states UTF-8 encoding, `ROS<char>` states UTF-16 encoding, and the string overload doesn't specify at all.

For the string case, I don't specify the encodings, but kept it for the `ROS<char>` one.

https://github.com/dotnet/corefx/blob/27dae83598c87e3cf4b139c8c981c13fe8e9a81e/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs#L54

https://github.com/dotnet/corefx/blob/27dae83598c87e3cf4b139c8c981c13fe8e9a81e/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs#L71

https://github.com/dotnet/corefx/blob/27dae83598c87e3cf4b139c8c981c13fe8e9a81e/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs#L257

Reply:
"UTF-16" really only applies to bytes.  With `char` it's relevant where surrogates are concerned, but in that case `char` _means_ UTF-16... so mentioning the encoding is unnecessary to the point that it's probably confusing.

I strongly advocate not having it for the `ROS<char>`, which is why it does not currently have it.

Reply:
OK, I will fix it here for consistency. I filed an issue for the writer APIs, though I am not fully convinced that it is necessary. Presumably, this applies to summary and param docs as well, correct?

https://github.com/dotnet/corefx/issues/37794